diff --git a/hdl/milan/milan_datapath.sv b/hdl/milan/milan_datapath.sv
index ee15148f2..3074b46cc 100644
--- a/hdl/milan/milan_datapath.sv
+++ b/hdl/milan/milan_datapath.sv
@@ -7169,7 +7169,7 @@ module milan_datapath import ethernet_packet_pkg::*; #(
     .clk_i (axis_clk), .rst_n (axis_resetn),
     .enable_i (cfg_maap_enable),
     .port_operational_i (eff_link_w),
-    .realtime_ns_i     (ptp_now_w[31:0]),
+    .realtime_ns_i     (32'd0),
     .count_i  (cfg_maap_count),
     .station_mac_i ({cfg_mac_addr[7:0],   cfg_mac_addr[15:8],
                      cfg_mac_addr[23:16], cfg_mac_addr[31:24],
