--- base
+++ head
@@ -42,14 +42,14 @@
 YOSYS OK  KL_srp_top
 YOSYS OK  protocol_processor_top
 YOSYS 42 tops, all.v parsed 1 time(s)
-Warning: Replacing memory \tpend_deadline_r with list of registers. See all.v:3651
-Warning: Replacing memory \adv_st_r with list of registers. See all.v:3649
-Warning: Replacing memory \lost_cnt_r with list of registers. See all.v:13336
-Warning: Replacing memory \payload_r with list of registers. See all.v:13335
-Warning: Replacing memory \age_r with list of registers. See all.v:16183
-Warning: Replacing memory \tf_cnt_r with list of registers. See all.v:20548
-Warning: Replacing memory \tf_q_r with list of registers. See all.v:20477
-Warning: Replacing memory \tf_wptr_r with list of registers. See all.v:20544, all.v:20470
+Warning: Replacing memory \tpend_deadline_r with list of registers. See all.v:3667
+Warning: Replacing memory \adv_st_r with list of registers. See all.v:3665
+Warning: Replacing memory \lost_cnt_r with list of registers. See all.v:13352
+Warning: Replacing memory \payload_r with list of registers. See all.v:13351
+Warning: Replacing memory \age_r with list of registers. See all.v:16199
+Warning: Replacing memory \tf_cnt_r with list of registers. See all.v:20564
+Warning: Replacing memory \tf_q_r with list of registers. See all.v:20493
+Warning: Replacing memory \tf_wptr_r with list of registers. See all.v:20560, all.v:20486
 Warning: Resizing cell port KL_aecp_engine.u_ucpu.rom_r.0.2.DOPBDOP from 8 bits to 4 bits.
 Warning: Resizing cell port KL_aecp_engine.u_ucpu.rom_r.0.2.DOBDO from 64 bits to 32 bits.
 Warning: Resizing cell port KL_aecp_engine.u_ucpu.rom_r.0.2.DOPADOP from 8 bits to 4 bits.
