--- head/hdl/srp/KL_srp_top.sv	2026-10-04 14:31:45.000000000 +0200
+++ ctl/tf-tk-write-at-rptr/srp/KL_srp_top.sv	2026-10-04 14:45:29.920852597 +0200
@@ -965,7 +965,7 @@
 
   always_ff @(posedge clk_i) begin : tf_tk_write
     if (tf_push_w[0]) begin
-      tf_tk_ram_r[tf_wptr_r[0]]
+      tf_tk_ram_r[tf_rptr_r[0]]
         <= {tk_arm_cancel_w, tk_arm_slot_w, tk_arm_owner_w, tk_arm_dl_w};
     end
   end
