--- $SCRATCH/r1b/lockstep/head-3ab2e4da.sv	2026-10-04 00:27:22.396000000 +0200
+++ $SCRATCH/work/lockstep/mutf/last_chunk_ignored.sv	2026-10-03 18:02:08.040327071 +0200
@@ -589,7 +589,7 @@
       end
       assign ch_w[j] = mem_r[ix_key_w[j*IXC_W_C +: IXC_W_C]];
     end
-    assign ix_hit_w[i] = &ch_w;
+    assign ix_hit_w[i] = &ch_w[N_IXC_C-2:0];
   end
 
   always_comb begin : command_registry_hit
