--- $REVIEWS/pp230-r458-4-packet/scratch/run-head/hdl/srp/KL_srp_top.sv	2026-10-04 16:34:26.211000000 +0200
+++ $REVIEWS/pp230-r458-4-packet/scratch/tmp/r458-4-vr1crdm3/hdl/srp/KL_srp_top.sv	2026-10-04 16:52:34.442564087 +0200
@@ -973,7 +973,7 @@
   always_ff @(posedge clk_i) begin : tf_ls_write
     if (tf_push_w[1]) begin
       tf_ls_ram_r[tf_wptr_r[1]]
-        <= {ls_arm_cancel_w, ls_arm_slot_w, ls_arm_owner_w, ls_arm_dl_w};
+        <= {1'b0, ls_arm_slot_w, ls_arm_owner_w, ls_arm_dl_w};
     end
   end
 
